src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
```

Comment:
if you like, you can pull the `transaction` off of props to avoid some repetition. `const { transaction } = props`. Or if you really want to go crazy `const {transaction: { posted_at, description, category, amount} = props`